Makefile

```diff
@@ -10,16 +10,20 @@ download:
 	bash publish/download.sh
 
 .PHONY: test
-test: test-stop
-	bash test/test_suite.sh ${SUITE}
+test: 
```

Comment:
Doesn't it still need to depend on `test-stop`?

Reply:
It doesn't anymore because of these changes @roti made awhile back which stops, and removes the `test` containers and volumes before each test run: https://github.com/wmde/wikibase-release-pipeline/blob/244f2e363ab7d8f27f1a64eb4e591bde3e92d701/test/run_selenium.sh#L34C15-L36

`test_stop.sh` can and probably should be removed for clarity though.

Reply:
Should we remove `test-stop.sh` and the make targets in this PR then?

Reply:
I was being cautious, but have went through it on this and will remove in this branch as well.